projects/1.16/assets/hanami/hanami/lang/zh_cn.json

```
@@ -1 +1,65 @@
-{}
\ No newline at end of file
+{
+    "_comment": "樱木类",
+    "block.hanami.stripped_sakura_log": "去皮樱花原木",
+    "block.hanami.sakura_log": "樱花原木",
+    "block.hanami.stripped_sakura_wood": "去皮樱花木",
+    "block.hanami.sakura_wood": "樱花木",
+    "block.hanami.sakura_planks": "樱花木板",
+    "block.hanami.sakura_slab": "樱花木台阶",
+    "block.hanami.sakura_stairs": "樱花木楼梯",
+    "block.hanami.sakura_pressure_plate": "樱花木压力板",
+    "block.hanami.sakura_fence": "樱花木栅栏",
+    "block.hanami.sakura_fence_gate": "樱花木栅栏门",
+    "block.hanami.sakura_button": "樱花木按钮",
+    "block.hanami.sakura_trapdoor": "樱花木活板门",
+    "block.hanami.sakura_door": "樱花木门",
+    "block.hanami.sakura_sign": "樱花木告示牌",
+    "block.hanami.sakura_leaves": "樱花树树叶",
+    "block.hanami.sakura_sapling": "樱花树树苗",
+    "block.hanami.sakura_leaf_carpet": "樱花叶地毯",
+    "block.hanami.sakura_bookshelf": "樱花木书架",
+    "block.hanami.sakura_chest": "樱花木箱子",
+    "block.hanami.sakura_trapped_chest": "樱花木陷阱箱",
+    "block.hanami.sakura_ladder": "樱花木梯子",
+    "block.hanami.sakura_vertical_slab": "樱花木竖直台阶",
+    "block.hanami.vertical_sakura_planks": "竖纹樱花木板",
+    "block.hanami.sakura_beehive": "樱花蜂箱",
+    "block.hanami.potted_sakura_sapling": "樱花树树苗盆栽",
+    "item.hanami.sakura_boat": "樱花木船",
+    "_comment2": "食物",
+    "effect.hanami.instability": "失稳",
+    "effect.hanami.nourishing": "滋润",
+    "effect.hanami.unsalutary": "无益",
+    "item.hanami.sakura_cherries": "樱桃",
+    "item.hanami.sakura_cherry_juice": "樱桃汁",
+    "item.hanami.sakura_cherry_kombucha": "樱花康普茶",
+    "item.hanami.sakura_cherry_pie": "樱桃派",
+    "block.hanami.black_forest_gateau": "黑森林蛋糕",
+    "block.hanami.sakura_cherry_sack": "樱桃袋",
+    "block.hanami.cherry_sakura_leaves": "樱桃叶",
+    "_comment3": "群系",
+    "biome.hanami.sakura_forest": "樱花森林",
+    "biome.hanami.sakura_forest_edge": "樱花森林边缘",
+    "biome.hanami.sakura_plateau": "樱花平原",
+    "biome.hanami.sakura_rolling_hills": "樱花山岗",
+    "biome.hanami.sakura_valley": "樱花山谷",
```

```suggestion
    "biome.hanami.sakura_valley": "樱花峡谷",
```
原版`valley`用的峡谷 但其实问题不大 https://minecraft.fandom.com/zh/wiki/%E7%94%9F%E7%89%A9%E7%BE%A4%E7%B3%BB?variant=zh